pysqldb3/geopackage.py

```diff
@@ -422,52 +422,52 @@ def read_gpkg(self, precision=False, private=True, gpkg_encoding=None, print_cmd
 
             # if the bulk upload involves only 1 table and no db table name was specified, we name the table the same thing as the original gpkg table
```

Comment:
seems like a lot of the same code from gpkg to shp, can you move to an internal function to DRY this up?

Reply:
I have cleaned it up in my next update